OWNERS_ALIASES

```diff
@@ -0,0 +1,18 @@
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
```

Comment:
https://go.k8s.io/owners is a more convenient link

anyhow, this is incorrectly formatted, these need to be under an `aliases:` top level key.

Reply:
🤦  thanks for catching this!